### PR TITLE
Add Theme and Edit Login Page

### PR DESCRIPTION
**Pull request by karlpandacan**

### CODE REVIEW

File: resources/views/layouts/app.blade.php [hunk 1]
```diff
@@ -12,49 +12,37 @@
     <link href="https://fonts.googleapis.com/css?family=Lato:100,300,400,700" rel='stylesheet' type='text/css'>
 
     <!-- Styles -->
-    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet">
-    {{-- <link href="{{ elixir('css/app.css') }}" rel="stylesheet"> --}}
+    {{--<link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet">--}}
```
Comment (nguyenthanhtung88) [inline resources/views/layouts/app.blade.php]: Oops, you didn't use bootstrap css from CDN right?
And you decided to copy/paste all content from bootstrap original file to `app.css` file?
I think it's not good, you should import original bootstrap file and make your customization at `app.css` file.

Comment (karlpandacan) [inline resources/views/layouts/app.blade.php]: @nguyenthanhtung88 Yes I did not use the CDN bootstrap, but I did not copy paste my design directly to public\css\app.css, I generated the design creating/editing our own LESS file and generating it thru resource\assets\sass\app.scss and gulp.

Comment (nguyenthanhtung88) [inline resources/views/layouts/app.blade.php]: @karlpandacan : That's ok but i think your solution is not good when you tried to edit bootstrap original less file. What will happen if next time bootstrap release a new version and what will you do to apply that version to your application?
My suggestion is you should respect core code of bootstrap, import original file to your application, and if you want to customize any class of bootstrap, write style custom to your `app.scss` file (we can think it override default class of bootstrap).
The advantages is:
1. If you want to upgrade bootstrap version, put a new file override original bootstrap, your custom style still existed in `app.scss`.
2. If you don't need custom style, comment code in `app.scss` file and you can use original version of bootstrap for that style.

P/S: Your solution is still ok, if you don't want to make any changes, that's no problem.
But i just want to give you best practices when working with third-party library :smiley:

**Comment by dinhhoanglong91** [comment]
ok

**Pull request merged**